release

```diff
@@ -58,5 +58,17 @@ fi
 
 git fetch
 git checkout "$COMMITISH"
-git tag "${DEPLOYMENT}-$(date --iso-8601=minutes)"
+
+if [ "$(uname)" == "Darwin" ]; then
+  which -s brew
+  if [[ $? != 0 ]] ; then
+    # Install Homebrew
+    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+  fi
+  brew install coreutils
```

Comment:
I don't think installing gdate should be part of the script.

I'd prefer just an error message telling the user how to do it.